### PR TITLE
DP-2472 Revert "diff commit (#10)"

### PR DESCRIPTION
**Pull request by martha-bighealth**
This reverts commit e7a11f0407b7f8d3d10a5c41e3782acd5e2c2b08.

https://bighealth.atlassian.net/browse/DP-2472

It will add the changes that Luca reverted back so that we can move forward with the Android CC changes.

Thanks for opening a PR!
Since this is a volunteer project and is very active, anything you can do to reduce the amount of time needed to review and merge your PR is appreciated.
The following steps will help get your PR merged quickly:

#### Update the documentation
If you've added new functionality, update the README.md with an entry for your prop or event.
The entry should be inserted in alphabetical order.

#### Update the changelog
After you open the PR, update the CHANGELOG.md file with an entry pointing to your PR.

#### Provide an example of how to test the change
If the PR requires special testing setup provide all the relevant instructions and files. This may include a sample video file or URL, configuration, or setup steps.

#### Focus the PR on only one area
Testing multiple features takes longer than isolated changes and if there is a bug in one feature, prevents the other parts of your PR from getting merged until it gets fixed.
If you're touching multiple different areas that aren't related, break the changes up into multiple PRs.

#### Describe the changes
Add a note describing what your PR does. If there is a change to the behavior of the code, explain why it needs to be updated.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,10 +1,83 @@
...
+- Fix being unable to disable sideloaded texttracks in the AVPlayer [#2679](https://github.com/react-native-video/react-native-video/pull/2679)
+- Fixed crash when iOS seek method called reject on the promise [#2743](https://github.com/react-native-video/react-native-video/pull/2743)
+- Fix maxBitRate property being ignored on Android [#2670](https://github.com/react-native-video/react-native-video/pull/2670)
+- Fix crash when the source is a cameraroll [#2639] (https://github.com/react-native-video/react-native-video/pull/2639)
+- Fix IOS UI frame drop on loading video [#2848] (https://github.com/react-native-video/react-native-video/pull/2848)
+
+### Version 6.0.0-alpha.1
+
+- Remove Android MediaPlayer support [#2724](https://github.com/react-native-video/react-native-video/pull/2724)
+  **WARNING**: when switching from older version to V6, you need to remove all refrerences of android-exoplayer. This android-exoplayer folder has been renamed to android. Exoplayer is now the only player implementation supported.
```
Comment (ruby-bighealth) [inline CHANGELOG.md:53]: nit: not a blocker 
Typo - *references

File: examples/basic/android/app/src/main/java/com/videoplayer/MainApplication.java [hunk 2]
```diff
@@ -1,51 +1,91 @@
...
-        new ReactVideoPackage(),
-        new RNCPickerPackage()
-      );
-    }
+        @Override
+        protected List<ReactPackage> getPackages() {
+          @SuppressWarnings("UnnecessaryLocalVariable")
+          List<ReactPackage> packages = new PackageList(this).getPackages();
+          // Packages that cannot be autolinked yet can be added manually here, for example:
+          // packages.add(new MyReactNativePackage());
```
Comment (ruby-bighealth) [inline examples/basic/android/app/src/main/java/com/videoplayer/MainApplication.java:30]: Observation: (not a blocker) For some reason these two lines don't look like they are commented out? Just assuring that they are.

**Review by ruby-bighealth — APPROVED** [review]
Looks good to me! 

Have some non-blocking comments

File: examples/basic/android/app/src/main/java/com/videoplayer/MainApplication.java (hunk 2 above)
Comment (ruby-bighealth) [reply examples/basic/android/app/src/main/java/com/videoplayer/MainApplication.java:30]: Oh nvm! In the conversation view, they look commented out. 

Pls disregard this - This is what it looked like when I initially commented 
Screen Shot 2023-06-06 at 1 45 11 PM

**Comment by martha-bighealth** [comment]
@ruby-bighealth all of those typos and comments come directly from the react native video player main fork. The only changes in our branch that come from us were from 3 years ago.

**Review by taber-bighealth — APPROVED** [review]
PR is an update and changes are too numerous to review all at once. Since changes are from an externally maintained library I shall approve it and let QA ensure functionality.

**Review by peter-casanova-bighealth — APPROVED** [review]
This PR requires regression testing.  Too many files changed for a proper code review.

**Pull request merged**